src/frontend/app/features/service-catalog/service-summary/service-summary.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="summary">
+  <app-tile-grid fit="true">
+    <app-tile-group>
+      <app-tile>
+        <app-service-summary-card></app-service-summary-card>
+      </app-tile>
+      <app-tile>
+        <app-service-broker-card></app-service-broker-card>
+      </app-tile>
+    </app-tile-group>
+    <app-tile-group>
+      <app-tile>
+        <app-card-number-metric [link]="library_books" icon="cloud" label="Instances" labelSingular="Instance" value="{{ (instances$ | async)?.length }}"></app-card-number-metric>
```

Comment:
On some of the cf pages these are links. For this one we can link to the instances tab